torch/distributed/rpc/constants.py

```diff
@@ -1,12 +1,16 @@
-from datetime import timedelta
 from torch.distributed.constants import default_pg_timeout
 
-# For any RpcAgent.
-DEFAULT_RPC_TIMEOUT = timedelta(seconds=60)
-DEFAULT_INIT_METHOD = "env://"
+from . import (
+    _DEFAULT_RPC_TIMEOUT,
```

Comment:
Is this change because they are defined in C++ and exposed to python now?

Reply:
Yes. We need a default value for the `ProcessGroupRpcBackendOptions` constructor arguments, so that these defaults need to live in C++. Important them here to avoid duplications.